src/internal/task/task.go

```diff
@@ -39,6 +39,8 @@ type Service interface {
 	NewSource(namespace string) Source
 	// TaskCount returns how many tasks are in a namespace and how many are claimed.
 	TaskCount(ctx context.Context, namespace string) (tasks int64, claims int64, _ error)
+	// ListTaskF calls a function on every task under a given namespace/etcd prefix
+	ListTaskF(ctx context.Context, prefix string, f func(key string, data *Task, claimed bool) error) error
```

Comment:
We have been moving away from the `F` suffix for callback based functions. We should just have this be `ListTask`. Also, we tend to use `cb` for the variable name of the callback function.

Reply:
It might actually make sense to just call this `List` since task is implicit in the package.

Reply:
Also, we should be able to remove the `TaskCount` method from the interface. We could move that to a utility function based on the `List` method. See the `util.go` file for an example using the `Do` method.